src/hooks/useDittoSingleText.ts

```diff
@@ -1,27 +1,31 @@
 import { useContext } from "react";
-import { DittoContext, SourceDetector } from "../lib/context";
-import { nullError } from "../lib/utils";
+import { Plurals } from "../components/Ditto";
```

Comment:
I think we should add fallback values for `variables` instead of relying on the default argument in this file, for reasons described [here](https://github.com/dittowords/ditto-react/pull/10/files#r784865890)